scanners/wpscan/values.yaml

```diff
@@ -1,6 +1,6 @@
 parserImage:
   registry: docker.io
-  repository: scbexperimental/parser-wpscan
+  repository: scbexperimental/wpscan-parser
```

Comment:
Would prefer to stick with scbexperimental/parser-wpscan to be consistent with the nameing of the other parser images